openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -137,10 +138,23 @@ def limits_change_callback(packet, item, old_limits_state, value, log_change)
       packet_time = packet.packet_time
       if value
         message = "#{packet.target_name} #{packet.packet_name} #{item.name} = #{value} is #{item.limits.state}"
+        if item.limits.values
+          values = item.limits.values[System.limits_set]
+          # Check if the state is RED_LOW, YELLOW_LOW, YELLOW_HIGH, RED_HIGH, GREEN_LOW, GREEN_HIGH
+          if LIMITS_STATE_INDEX[item.limits.state]
+            # Directly index into the values and return the value
+            message += " (#{values[LIMITS_STATE_INDEX[item.limits.state]]})"
+          elsif item.limits.state == :GREEN
+            # If we're green we display the green range (YELLOW_LOW - YELLOW_HIGH)
+            message += " (#{values[1]} to #{values[2]})"
+          elsif item.limits.state == :BLUE
+            # If we're blue we display the blue range (GREEN_LOW - GREEN_HIGH)
+            message += " (#{values[4]} to #{values[5]})"
+          end
+        end
       else
         message = "#{packet.target_name} #{packet.packet_name} #{item.name} is disabled"
       end
-      message << " (#{packet.packet_time.sys.formatted})" if packet_time
```

Comment:
Removed the time because it's already in the table

Reply:
I'm unsure about not including packet_time.   Received time and packet time are not necessarily the same.  Especially true for stored values (which should still alert).